src/email/templates/UnvalidatedEmailResetPassword/UnvalidatedEmailResetPasswordText.ts

```diff
@@ -0,0 +1,14 @@
+export const UnvalidatedEmailResetPasswordText = () => `
+Hello,
+Because your security is extremely important to us, we have changed our password policy. For this reason, you need to reset your password.
```

Comment:
```suggestion
Hello,

Because your security is extremely important to us, we have changed our password policy. For this reason, you need to reset your password.
```